airqo-platform/AirQo-api
# separating ENVS for the datawarehouse queries

**_WHAT DOES THIS PR DO?_**
separating ENVS for the datawarehouse queries

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [PLAT-1188]

**_HOW DO I TEST OUT THIS PR?_**
 `cd src/device-registry`

- check README for run instructions
- Please ensure that you get the [latest ENVs](https://airqo.slack.com/archives/C03218Z31JT/p1657721465851329) in your .envs.

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

- [x] [Get Events version 2](https://app.gitbook.com/o/-MCogOVGQqjUharJKchL/s/-MKqyQkcfs54GYi-q96G/device-registry/events#version-2-get-events)




[PLAT-1188]: https://airqoteam.atlassian.net/browse/PLAT-1188?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
